Core/src/main/java/com/theprogrammingturkey/comz/util/CommandUtil.java

```diff
@@ -1,6 +1,11 @@
 package com.theprogrammingturkey.comz.util;
 
 import com.theprogrammingturkey.comz.COMZombies;
+
+import me.clip.placeholderapi.libs.kyori.adventure.platform.viaversion.ViaFacet.Chat;
+import net.md_5.bungee.api.chat.*;
```

Comment:
Haven't seen this code space before. Just want to double check that this doesn't put any extra restrictions on who can run this plugin

Reply:
Normally the first import was an unused or wrong imported one, the other normally is right, it's is for the clickable text.

Reply:
For context, net.md_5.bungee.api.chat package is added in the runable spigot jar itself, even tho bungeecord is the root package.

Reply:
Seems both Spigot and PaperMC support this. Good enough for me